libs/template/src/lib/push/push.pipe.ts

```diff
@@ -64,13 +64,13 @@ import { filter, switchMap, tap, withLatestFrom } from 'rxjs/operators';
  * @publicApi
  */
 @Pipe({ name: 'push', pure: false })
-export class PushPipe<U> implements PipeTransform, OnDestroy {
+export class PushPipe implements PipeTransform, OnDestroy {
   /** @internal */
-  private renderedValue: U | null | undefined;
+  private renderedValue: any | null | undefined;
```

Comment:
can we add a comment here to explain at least why this is any? I could imagine that we will introduce some linting rule which will complain about this in the future.

Reply:
Without a class-level generic type option, there is no longer a way to know what value type will be passed into the `transform()` method. Happy to leave a code comment about that if you think it's necessary, but may be better just for the issue notes since it should be generally inferable?

Reply:
I understand why we change it to any. just some internal note for us as reminder :). u think its worthless?

Reply:
Oh I see. If that's your intent sure! Sorry, I'm not used to using comments for this purpose. I'll update it.

Reply:
love it